components/Header.tsx

```diff
@@ -33,7 +33,7 @@ export function Header() {
 
   return (
     <header>
-      <Link href="/">
```

Comment:
I don't think this is needed here as the `h1` does not have a `href` prop
<img width="165" alt="image" src="https://user-images.githubusercontent.com/16056918/198985935-88998b5a-ff96-489b-b762-e475cc8537f3.png">